src/parsec_service/tests/existing_tests.md

```diff
@@ -0,0 +1,55 @@
+# List of existing tests
+
+## Integration tests on providers
+
+| Name                                                                                          | Description                                                                                                                                                                                                                                                                             |
+|-----------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [TPM integration tests](https://github.com/parallaxsecond/parsec/tree/master/tests/providers) | These tests initialize a provider structure and perform operation using the `Provide` trait. A software crypto library, like `ring`, can be used to verify the behaviour. [Example with the TPM provider](https://github.com/parallaxsecond/parsec/blob/master/tests/providers/tpm.rs). |
+
```

Comment:
Sorry to bug you with this, but I'm moving these to the E2E tests 😬  we could do it either way - you change this or  change it after mine gets merged.

Reply:
I guess you can just say in the "Normal tests" line below that we also verify Parsec results with 3rd party software implementations of crypto (`ring`, the `rsa` crate...)